lib/cyclonedx/report_podfile_lock_cyclonedx.rb

```diff
@@ -0,0 +1,10 @@
+module Cyclonedx
+  class ReportPodfileLock < Base
+    TYPE = "cocoa".freeze
```

Comment:
It appears that this `TYPE` is not used?

Reply:
[It's used by the base class.](https://github.com/coinbase/salus/blob/8d4d7c8f80a0087b3e1fd1302be356d93369d842/lib/cyclonedx/base.rb#L42)